src/inabox/inabox-viewport.js

```diff
@@ -177,9 +180,14 @@ export class ViewportBindingInabox {
   /** @override */
   getLayoutRect(el) {
     const b = el./*OK*/getBoundingClientRect();
+    let {left, top} = b;
+    if (this.useLayers_) {
+      left -= this.viewportRect_.left;
```

Comment:
can you explain why do we offset with viewportRect?

Reply:
oh, is layer's coordinate system relative to viewport instead of doc?

Reply:
Yes, Layer’s rects should exactly match `getBoundingClientRect` (with rounding) in every circumstance.

In this case, we need to factor in that the is relative to a viewport in a parent document.